packages/ui/components/calendar/test/lion-calendar.test.js

```diff
@@ -465,9 +465,8 @@ describe('<lion-calendar>', () => {
         `);
 
         clock.restore();
-        expect(isSameDate(el.centralDate, new Date('2019/06/03')), 'central date').to.be.true;
-        expect(isSameDate(elSetting.centralDate, new Date('2019/07/03')), 'central date').to.be
-          .true;
+        expect(isSameDate(el.centralDate, new Date('2019/07/03'))).to.be.true;
```

Comment:
After the PR, the central date would be nearest enabled date instead of today even when the date is disabled, which is the desired behavior, I believe.